r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -50,49 +50,70 @@ test_df <- tibble::tibble(
 test_that("strptime", {
   t_string <- tibble(x = c("2018-10-07 19:04:05", NA))
   t_stamp <- tibble(x = c(lubridate::ymd_hms("2018-10-07 19:04:05"), NA))
+  t_stamp_with_pm_tz <- tibble(
+    x = c(lubridate::ymd_hms("2018-10-07 19:04:05", tz = "Pacific/Marquesas"), NA)
+  )
+
+  withr::with_timezone("Pacific/Marquesas", {
```

Comment:
Please can you chuck in a comment above this set of tests as to why we're using `expect_equal()` and not `compare_dplyr_binding()`?

Reply:
Done.